sql/sys_vars.cc

```diff
@@ -3889,6 +3889,7 @@ static const char *old_mode_names[]=
   "COMPAT_5_1_CHECKSUM",                // deprecated since 11.3
   "NO_NULL_COLLATION_IDS",              // deprecated since 11.3
   "LOCK_ALTER_TABLE_COPY",              // deprecated since 11.3
+  "SESSION_USER_IS_USER",
```

Comment:
when you'll rebase it on top of 11.6, you'll see a conflict here, see how the new old mode `OLD_FLUSH_STATUS` was added and add yours in the same way with a correct deprecation version.